src/main.rs

```diff
@@ -38,6 +38,9 @@ use embedded_hal::blocking::delay::DelayMs;
 
 use crate::hal::spi::Enabled;
 
+use heapless::Vec;
```

Comment:
```suggestion
```

Reply:
Resolved in [following commit](https://github.com/Jim-Hodapp-Coaching/esp32-pico-wifi/pull/4/commits/c10bb540c941bdb44dbff93ce28af3736f1ca7b1)